include/fprotect.h

```diff
@@ -65,6 +66,7 @@ int fprotect_area_no_access(uint32_t start, size_t length);
  * @retval 3   If write and read protected
  */
 uint32_t fprotect_is_protected(uint32_t addr);
+#endif /* CONFIG_HAS_HW_NRF_ACL */
```

Comment:
This will break use of `IS_ENABLED` and is discouraged per zephyr coding standards, only function _definitions_ should be `ifdef`-ed out.

https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html#rule-a-1-conditional-compilation

Reply:
fixed. Thank you, I forgot about how this worked.